### PR TITLE
feat(@wallet): Sort account by position

### PR DESCRIPTION
**Pull request by alaibe**
#11133

status-go: https://github.com/status-im/status-go/pull/3645

### CODE REVIEW

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM!

**Review by stefandunca — APPROVED** [review]

File: ui/app/AppLayouts/Profile/views/wallet/AccountOrderView.qml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import QtQuick 2.15
+import SortFilterProxyModel 0.2
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Profile/views/wallet/AccountOrderView.qml:2]: some of these imports are not used and can be removed

File: ui/app/AppLayouts/Profile/views/wallet/AccountOrderView.qml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            anchors {
+                horizontalCenter: parent.horizontalCenter
+                verticalCenter: parent.verticalCenter
+            }
+
+            dragParent: accountsView
+            visualIndex: delegateRoot.visualIndex
+            draggable: accountsView.count > 1
+            title: {
+                console.log(model.name, model.position)
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Profile/views/wallet/AccountOrderView.qml:89]: leftover

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM just one leftover line of code

**Pull request merged**